src/npc.h

```diff
@@ -742,20 +741,25 @@ class npc : public player
      * This does not change the global position of the NPC.
      */
     tripoint global_square_location() const override;
- int plx, ply, plt;// Where we last saw the player, timeout to forgetting
- int itx, ity; // The square containing an item we want
- int guardx, guardy;  // These are the local coordinates that a guard will return to inside of their goal tripoint
+    tripoint last_player_seen_pos; // Where we last saw the player
+    int last_seen_player_turn; // Timeout to forgetting
```

Comment:
It is currently disabled, but I left it in for saves (and eventual re-enabling).
It should be timeout, yes - I'll rename it when adding the functionality.